prac_05/state_names.py

```diff
@@ -6,7 +6,7 @@
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 
-# Printing all states and names
+# Printing states and names
 for code, name in CODE_TO_NAME.items():
     print(f"{code} is {name}")
```

Comment:
String formatting could be implemented to neatly line up the state codes in output.